yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtElement.kt

```diff
@@ -32,7 +32,7 @@ internal abstract class CrdtElement(
 
     var removedAt: TimeTicket? = removedAt
         set(value) {
-            if (value > createdAt && value > field) {
+            if (value >= createdAt && value > field) {
```

Comment:
@skhugh this condition has some issues when setting dummyHead in RgaTreeList, so I fixed it.

Reply:
I actually made custom setter to replace `remove` function.
we should delete one of them.